speckle/base/resource.py

```diff
@@ -49,8 +51,8 @@ def clean_empty(d):
     if not isinstance(d, (dict, list)):
         return d
     if isinstance(d, list):
-        return [v for v in (clean_empty(v) for v in d) if v]
-    return {k: v for k, v in ((k, clean_empty(v)) for k, v in d.items()) if v}
+        return [v for v in (clean_empty(v) for v in d) if v != None]
```

Comment:
I think using `v is not None` is considered more `pythonic`

Reply:
Right-eo. I agree, if there's a better / easier way of writing queries, let's do that :P but it works for simple stuff for now!

Reply:
Changed to `is not`.